src/components/ModalBox/ModalBox.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <div class="box-wrapper box-alert-mask" @click="toggle">
+        <div class="box-alert-wrapper">
+            <slot></slot>
+        </div>
+    </div>
+</template>
+
+<script>
+    export default {
+        name: 'ModalComponent',
+        components: {
+        },
+        props: {
+            toggle: { type: Function ,required:true},
+        }
+    }
+
+</script>
+
+<style scoped>
+    .box-wrapper {
+        width: 100%;
+        height: 100%;
+        min-height: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content:center
+    }
+
```

Comment:
there are some linting issues, can you fix these? `yarn lint --fix` can fix all the files for you :)

Reply:
it's showing me that no lint errors were found.

> there are some linting issues, can you fix these? `yarn lint --fix` can fix all the files for you :)

it's showing me that no lint errors were found.

Reply:
![image](https://user-images.githubusercontent.com/55620053/180616938-50e25f3d-ba63-41e0-8624-3225961ad1c4.png)

Reply:
@Janvi-Thakkar Can we try having the .prettierrc and .eslintrc.js files from the FLINT-UI repo in this repo as well? 
Maybe the linting issues are not detected because these files are not present.